parser.js

```diff
@@ -61,10 +58,10 @@ function parseProgram() {
     error('KobraScript programs may not be empty')
   }
 
-  var initialBlock = new Block(parseStatements())
+  var programBlock = new Block(parseStatements())
   match('EOF')
```

Comment:
Yes! This is more explicit for what it is: the block that represents the highest level, which is the program